src/base/utils/net.h

```diff
@@ -28,13 +28,23 @@
 
 #ifndef BASE_UTILS_NET_H
 #define BASE_UTILS_NET_H
+
 class QString;
 
 namespace Utils
 {
     namespace Net
     {
         bool isValidIP(const QString &ip);
+
+        template <typename T>
+        T unquote(const T &str)
```

Comment:
>Have you ever thought that there is also error? I now think so.

As a string helper function yes, but no as a helper for http/net, only side effects.

oh geez, so much friction for a specific use helper function... alright I'll take @glassez code and ignore the case: `"` (didn't handle other invalid tokens anyway...)